addons/point_of_sale/models/pos_note.py

```diff
@@ -0,0 +1,9 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+class PosNote(models.Model):
+    _name = 'pos.note'
```

Comment:
```suggestion
    _name = 'pos.note'
    _description = 'PoS Note'
```
Runbot is complaining about it.